src/main/java/eu/ha3/presencefootsteps/sound/acoustics/ChanceAcoustic.java

```diff
@@ -5,30 +5,33 @@
 import eu.ha3.presencefootsteps.sound.State;
 import eu.ha3.presencefootsteps.sound.player.SoundPlayer;
 import net.minecraft.entity.LivingEntity;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Range;
+
+record ChanceAcoustic(
+        @NotNull Acoustic acoustic,
+        @Range(from = 1, to = Integer.MAX_VALUE) float probability
+) implements Acoustic {
+
+    @Contract(value = "_, _ -> new", pure = true)
```

Comment:
Remove @ Range and @ Contract annotations. Not sure what purpose they're going to serve as nothing I have makes use of them.

Reply:
I removed the Range annotation but for the Contract annotation I'm against removing it, I've pasted this from the docs:
_"Specifies some aspects of the method behavior depending on the arguments. Can be used by tools for advanced data flow analysis. Note that this annotation just describes how the code works and doesn't add any functionality by means of code generation."_